client/src/styles/main.css

```diff
@@ -160,12 +160,30 @@ body {
   opacity: 0.7;
 }
 
+.book-checkout-selected {
```

Comment:
`.book-checkout-selected` and `.book-checkin-selected` have the same css styles. this can be put into one